### PR TITLE
Update load_csv > load_csv_with_header for r 0.11

### PR DESCRIPTION
**Pull request by dmathewwws**

### CODE REVIEW

**Comment by dmathewwws** [comment]
I signed it!

File: tensorflow/g3doc/tutorials/tflearn/index.md [hunk 1]
```diff
@@ -35,11 +35,13 @@ IRIS_TRAINING = "iris_training.csv"
 IRIS_TEST = "iris_test.csv"
 
 # Load datasets.
-training_set = tf.contrib.learn.datasets.base.load_csv(filename=IRIS_TRAINING,
-                                                       target_dtype=np.int)
-test_set = tf.contrib.learn.datasets.base.load_csv(filename=IRIS_TEST,
-                                                   target_dtype=np.int)
-
+training_set = tf.contrib.learn.datasets.base.load_csv_with_header(filename=IRIS_TRAINING, 
+                                                       target_dtype=np.int,
```
Comment (rohan100jain) [inline tensorflow/g3doc/tutorials/tflearn/index.md]: indent?

File: tensorflow/g3doc/tutorials/tflearn/index.md [hunk 2]
```diff
@@ -35,11 +35,13 @@ IRIS_TRAINING = "iris_training.csv"
...
+                                                       features_dtype=np.float32)
```
Comment (rohan100jain) [inline tensorflow/g3doc/tutorials/tflearn/index.md]: from the old load_csv -> load_csv_with_header translation seems like this should perhaps be np.float64?

https://github.com/tensorflow/tensorflow/commit/2d4267507e312007a062a90df37997bca8019cfb#diff-8e7d29906b35612ebe283482505429d4

**Review by rohan100jain — CHANGES_REQUESTED** [review]

**Comment by terrytangyuan** [comment]
You also need to change the actual code that's being tested

**Comment by tegg89** [comment]
I also found that load_csv() needed to be changed. However, DNNClassifier.fit()'s input has also brought ValueError: metrics. I am still looking forward it.

**Comment by vrv** [comment]
Any updates @dmathewwws  ?

**Comment by vrv** [comment]
Closing to clear out backlog, ping this thread and we can reopen (or send a new PR after resolving conflicts).

**Pull request closed without merging**